geode-core/src/integrationTest/java/org/apache/geode/distributed/internal/membership/gms/messenger/JGroupsMessengerJUnitTest.java

```diff
@@ -289,27 +291,21 @@ public void testSerializationError() throws Exception {
       when(msg.getDSFID()).thenReturn((int) DataSerializableFixedID.HEARTBEAT_RESPONSE);
 
       // for code coverage we need to test with both a SerializationException and
-      // an IOException. The former is wrapped in a GemfireIOException while the
+      // an IOException. The former is wrapped in a MembershipIOException while the
       // latter is not
       doThrow(new SerializationException("")).when(msg).toData(any(DataOutput.class),
           any(SerializationContext.class));
-      try {
-        messenger.send(msg);
-        fail("expected a failure");
-      } catch (GemFireIOException e) {
-        // success
-      }
+      Set<?> failures = messenger.send(msg);
+      assertThat(failures).isNotNull();
```

Comment:
Looks like we changed the contract for messenger.send. Did all the callers get updated?

Reply:
It used to throw a runtime exception that we no longer have access to in membership.  Some of our code wasn't properly protected from that exception.